lib/resque/worker.rb

```diff
@@ -307,6 +327,8 @@ def kill_child
           log! "Child #{@child} not found, restarting."
           shutdown
         end
+      elsif @cant_fork && @current_job
```

Comment:
`cant_fork` is removed on master [here](https://github.com/al2o3cr/resque/pull/2/files#diff-693c73ae6c0943d273cdb8efc95ecec2L21) - TODO figure out what it was replaced with